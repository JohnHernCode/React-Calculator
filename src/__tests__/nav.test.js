import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Navbar from '../components/Nav';

describe('Nav component', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<Router><Navbar /></Router>);
    expect(tree).toMatchSnapshot();
  });
});
