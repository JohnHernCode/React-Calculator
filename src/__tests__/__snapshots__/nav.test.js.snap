// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component matches the snapshot 1`] = `
<div
  className="nav"
>
  <li
    className="logo"
  >
    Math Magician
  </li>
  <ul
    className="menu"
  >
    <li>
      <a
        className="item"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        className="item"
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li
      className="item"
    >
      <a
        className="nav-link"
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</div>
`;
